pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -97,8 +100,9 @@ func (s *Service) ReconcileLoadBalancer(openStackCluster *infrav1.OpenStackClust
 				ProtocolPort:   port,
 				LoadbalancerID: lb.ID,
 			}
+			mc := metrics.NewMetricPrometheusContext("lblistener", "create")
```

Comment:
```suggestion
			mc := metrics.NewMetricPrometheusContext("loadbalancer_listener", "create")
```
resource_subresource
That was the pattern we used in ccm.